# Request
geometry_msgs/Point bbox_real_pos

---

# Response
float32 score
float32 width
float32 height
float32 depth
geometry_msgs/Point position
geometry_msgs/Quaternion orientation